src/community/jdbcconfig/pom.xml

```diff
@@ -54,11 +54,6 @@
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
     </dependency>
-    <dependency>
-      <!-- blazing fast LZW compression/decompression -->
-      <groupId>com.ning</groupId>
-      <artifactId>compress-lzf</artifactId>
-    </dependency>
```

Comment:
This isn't really relevant to this PR but if you are going to remove the dependency in the pom, you should also remove it from the release assembly:
https://github.com/geoserver/geoserver/blob/master/src/community/release/ext-jdbcconfig.xml#L13

Reply:
good catch. Amended the commits to have both changes.